.github/ISSUE_TEMPLATE/3_technical_task.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 🔨 Technical task
+about: Task that is not necessarily user facing
+---
+
+<!-- A clear and concise description of what needs to be done.
+
+Include a Definition of Done: acceptance criteria and examples for this task.
+
+Include a Definition of Ready: possible unknowns that need to be addressed before starting to work on this task.
```

Comment:
'Related' includes a prerequisite, needs being made more visible and cover non-tickets?

Reply:
Will unify.